.env

```diff
@@ -70,4 +70,4 @@ R_TAG=latest
 DEVTOOLSET_VERSION=-1
 
 # Used for the manylinux and windows wheels
-VCPKG=c7e96f2a5b73b3278b004aa88abec2f8ebfb43b5
+VCPKG=fced4bef1606260f110d74de1ae1975c2b9ac549
```

Comment:
Had to bump vcpkg's version, otherwise I'm getting consistent download errors for mingw on windows.

Reply:
This hash points to the latest commit on vcpkg's main branch.